init.lua

```diff
@@ -189,4 +188,23 @@ torch.Tensor.isTensor = torch.isTensor
 -- remove this line to disable automatic heap-tracking for garbage collection
 torch.setheaptracking(true)
 
+function torch.multinomialAliasSetup(probs, state)
+    local prob_tbl, alias_tbl
```

Comment:
What if state[1] and state[2] are nil ? In that case torch.multinomialAliasSetup_() should only take `probs` as input.

i.e. `state[1], state[2] = torch.multinomialAliasSetup_(probs)`